translations/tds61.js

```diff
@@ -1154,6 +1155,10 @@ tds61 = {
                 {
                     tags[ge4meta[i]] = tds61.rules.ge4List[tags[ge4meta[i]]];
                 }
+                else if (tds61.rules.ge4List['ge:GENC:3:1-2:' + tags[ge4meta[i]]])
```

Comment:
Fix for fields that only have the three letter country code and not the full string.